src/main/java/ui/fragments/AllBoardsFragment.java

```diff
@@ -49,4 +50,8 @@ public List<String> getBoardsTitlesElements() {
         ElementsCollection allBoardsList = getAllBoardsInWorkspace();
         return ElementUtil.getListOfStrings(allBoardsList);
     }
-}
+
+    public void clickAllClosedBoards() {
+        rootElement().$x(VIEW_ALL_CLOSED_BOARDS).shouldBe(visible).click();
+    }
```

Comment:
```suggestion
    public SelenideElement getViewAllClosedBoardsButton() {
        return rootElement().$x(VIEW_ALL_CLOSED_BOARDS).shouldBe(interactable);
    }
```

Reply:
I can't do anything with that element except as click